training/README.md

```diff
@@ -0,0 +1,20 @@
+# Training script for our best model
```

Comment:
1) why is training folder is under tests folder? how are they related.. 

2) should not the data folder be under training? 
3) should not the notebooks folder be under this training? 

It feels to me, that you can bundle all of this under models folder as in:
models
|_ README.md 
|_ hybrid.pkl
|_ lexical.pkl
|_ lexical_plus.pkl
|_ semantic.txt 
|_training 
|__ data
|__ notebooks
|__ logs
|__ nl4opt_utils
|__ .py and .sh etc.

Reply:
I don't know why you see tests under training, I don't see it that way. But restructured according to your suggestions.